File: Learn_Android/getWallpaer/app/src/main/java/com/example/sunkai/getwallpaer/Tools/GlobalContext.java
package com.example.sunkai.getwallpaer.Tools;

import android.app.Application;
import android.content.Context;

/**
 * Created by sunkai on 2018/1/26.
 */

public class GlobalContext extends Application {
    public static Context context;

    @Override
    public void onCreate() {
        super.onCreate();
        context=getApplicationContext();
    }
}

File: Learn_Android/Old/MyApplication/app/src/main/java/com/example/sunkai/myapplication/Tools/GlobalContext.kt
package com.example.sunkai.myapplication.Tools

import android.app.Application
import android.content.Context

/**
 * Created by sunkai on 2018/1/26.
 */
class GlobalContext:Application(){
    companion object {
        lateinit var context:Context
    }

    override fun onCreate() {
        super.onCreate()
        context=applicationContext
    }
}